reference/compiling_for_android.rst

```diff
@@ -43,66 +44,12 @@ To set those environment variables on Windows, press Windows+R, type
 pane, then click on **Environment variables** on the window that
 appears.
 
-To set those environment variables on Linux, use
+To set those environment variables on Unix (e.g. Linux, Mac OSX), use
 ``export ANDROID_HOME=/path/to/android-sdk`` and
 ``export ANDROID_NDK_ROOT=/path/to/android-ndk``.
 
-Compiling
```

Comment:
In the end I removed this section altogether, as it's a duplicate of what comes later to build the export templates. As of now, one can't build the *editor* for Android, only the templates.